### ISSUE TITLE
DAC 24 - Medium Priority WCAG Level AA - Colour Contrast [Text] 01

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#24]
<img width="267" alt="DAC_Colour_Contrast_01" src="https://user-images.githubusercontent.com/75850106/155568636-0b01ffb0-d324-41a0-96c9-f1adc25c3474.png">

### Issue
There are names of places on the map that have a contrast ratio of less than the 4.5:1 minimum contrast. This means that low vision users may be unable to read the text.

Foreground: #EDEDED 
Background: #FFFFFF
The contrast ratio is: 1.2:1 
1.4.3 Contrast (Minimum) (AA)
Fail for large and regular text 1.4.6 Contrast (Enhanced) (AAA)
Fail for large and regular text

<img width="597" alt="DAC 24 Current code ref" src="https://user-images.githubusercontent.com/75850106/155568924-a425aedb-cf26-4947-9417-84d7ded9b396.png">

### Suggested Solution

Ensure that the contrast between text and the background is in accordance with WCAG 2.1 recommendations:

- If the text is not bold and its size is less than 18pt then it must meet a minimum colour contrast ratio of 4.5:1 to Pass AA.
- If the text is not bold and its size is at least 18pt then it must meet a minimum colour contrast ratio of 3:1 to Pass AA.
- If the text is bold and its size is less than 14pt then it must meet a minimum colour contrast ratio of 4.5:1 to Pass AA.
- If the text is bold and its size is at least 14pt then it must meet a minimum colour contrast ratio of 3:1 to Pass AA.
